pkg/fqdn/service/service.go

```diff
@@ -274,14 +274,14 @@ func (s *FQDNDataServer) sendAndRecvAckForDNSPolicies(stream pb.FQDNData_StreamP
 	// Build egress L7 DNS policies with endpoint information
 	var egressL7DnsPolicy []*pb.DNSPolicy
 	var identityToEndpointMapping []*pb.IdentityToEndpointMapping
-
+	var identityToPrefixMapping []*pb.IdentityToPrefixMapping
 	// Process identity to IPs mappings - build both for quick lookup map and endpoint mappings
 	identityIPMap := make(map[identity.NumericIdentity][]netip.Prefix)
 
 	for identityIP := range identityToIPs {
 		var prefixes []netip.Prefix
 		endpointToIPsBytes := make(map[uint64][][]byte) // Group IPs by endpoint ID for this identity
-
+		prefixWithoutEndpoint := [][]byte{}
```

Comment:
not sure we need this? we should send all prefix -> ID mappings, even if that prefix happens to correspond to a pod on the node.

Reply:
I still see prefixWithoutEndpoint?